content/en/product_analytics/charts/funnel_analysis.md

```diff
@@ -64,31 +61,76 @@ To combine an event, click the three dots next to an event and select **+ Combin
 
 Use the Group by dropdown to group the data by a specific attribute.
 
-**Note**: Group by does not work with the funnel steps visualization.
+**Note**: `Group by` does not work with the funnel steps visualization, your visualization automatically changes to a top list.
 
 ## Refine conversion
 
 You can further analyze the information on the funnel page to understand the conversion rate. Conversion rate is a crucial metric that measures the effectiveness of your site or application.
 
-You can analyze conversion by **session count** or **users**, which means you can understand how many sessions or users ended up completing the funnel.
+You can analyze conversion by **session**, **users** or **accounts**, which means you can understand conversion across all sessions, or by distinct users or accounts. This can be useful if you suspect, for instance, that a minority of your user base converts at a high rate.
+
+{{< img src="product_analytics/journeys/funnel_analysis/pana_funnel_refine_conversion.png" alt="Specify which facet you'd like to use to refine your conversion. The available options are Session, User, and Account." style="width:50%;" >}}
+
+
+<div class="alert alert-info"> 
+
+If you select <code>Session</code>, all steps must be completed within the same <code>@session.id</code> to count as a conversion.
```

Comment:
I would use bold to refer to dropdown selections throughout this topic rather than monospace. That way, it's harder for customers to get it confused with something they configured somewhere in their code.